### PR TITLE
add docs for PN instance type

### PR DESCRIPTION
**Pull request by gregolsky**

### CODE REVIEW

**Comment by gregolsky** [comment]
please review @PFYasu

**Review by PFYasu — APPROVED** [review]

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown [hunk 1]
```diff
@@ -141,24 +141,30 @@ number of IOPS reserved for your instances.
...
+Since this type of RavenDB clusters is designed to hold data of terrabyte order of magnitude, by default PN instance types run backups of type Snapshot to ensure fast data restore.
+{NOTE/}
 
 {NOTE: }
-The Performance production tier is **highly customizable**.  
-Contact Support to modify your server configuration, the number of nodes in your cluster, 
-and various parameters of their deployment.  
-If you're interested in high global availability, for example, a multi-region deployment 
-can be easily set and would probably suit you better than the default arrangement.  
+Local NVMe drives on PN clusters are encrypted by default using hardware-based encryption.
```
Comment (PFYasu) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown:162]: BTW do we want to add a note PN instance types are available in Azure and AWS?

Comment (gregolsky) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-instances.markdown:162]: Not at this time.

**Comment by ppekrol** [comment]
status?

**Comment by gregolsky** [comment]
It's done, need to merge and deploy after cloud deployment

**Pull request merged**